sdk/core/core-tracing/src/tracers/test/testTracer.ts

```diff
@@ -30,6 +31,37 @@ export interface SpanGraph {
   roots: SpanGraphNode[];
 }
 
+export interface TestTracerLike extends NoOpTracer {
```

Comment:
I'm confused. What is this interface used for?

Reply:
Weird, I didn't create this myself. I'll clean this up and see what's up.

Reply:
Oh I think I remember I did this while trying to fix something and I forgot to delete it.

Reply:
Done!